src/pages/App.tsx

```diff
@@ -76,35 +77,38 @@ export default function App() {
   }, [])
 
   return (
-    <Suspense fallback={null}>
-      <SkeletonTheme baseColor={theme.bg3} highlightColor={theme.bg2}>
-        <ApolloProvider client={subgraphClients[chainId as SWPRSupportedChains] || defaultSubgraphClient}>
-          <NetworkWarningModal />
-          <AppWrapper id="app-wrapper">
-            <HeaderWrapper>
-              <Header />
-            </HeaderWrapper>
-            <BodyWrapper>
-              <Web3ReactManager>
-                <SpaceBg>
-                  <Suspense fallback={<FallbackLoader />}>
-                    <Routes />
-                  </Suspense>
-                </SpaceBg>
-              </Web3ReactManager>
-              <Marginer />
-            </BodyWrapper>
-          </AppWrapper>
-          <ToastContainer
-            draggable={false}
-            className="custom-toast-root"
-            toastClassName="custom-toast-container"
-            bodyClassName="custom-toast-body"
-            position="top-right"
-            transition={Slide}
-          />
-        </ApolloProvider>
-      </SkeletonTheme>
-    </Suspense>
+    <>
+      <Helmet>{process.env.NODE_ENV !== 'production' && <meta name="robots" content="noindex, nofollow" />}</Helmet>
```

Comment:
This solution dont work since we deploy everywhere as produciton builds. 

Suggesion: Can move this to a seperate component so we can add more checks in future


Suggest to change to this

```suggestion
      <Helmet>
        {
          // eslint-disable-next-line no-restricted-globals
          (location.hostname.includes('swapr.netlify.app') || location.hostname.includes('swapr.site')) && (
            <meta name="robots" content="noindex, nofollow" />
          )
        }
      </Helmet>
```

Reply:
We should have 4 types of environments: Production, Development, Staging, Test. Netlify / Swapr should (maybe) run as staging in this case.

Reply:
I have set an environment variable `REACT_APP_BUILD_ENV` with value `staging` in netlify so you can use it

```suggestion
      <Helmet>
        {process.env.REACT_APP_BUILD_ENV === 'staging' && <meta name="robots" content="noindex, nofollow" />}
      </Helmet>
```

Reply:
ok, nice. What are the rest of the options? Prefer to add 'disable for all except production', not only staging

Reply:
Update PR with a new component and move title and description from index.html as an example. Waiting for info on the rest of REACT_APP_BUILD_ENV options, as I set it to `!== 'production'`. Is it correct now?

Reply:
All our builds swapr.site and PR builds are in netlify and they have this varaible. 
You can add this variable in CI for IPFS deploy also if you need.

Reply:
ok, it is fixed